blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -1,6 +1,6 @@
 import keymirror from 'keymirror';
 
-import fetch from 'isomorphic-fetch';
+import fetch from '../util/fetch';
```

Comment:
hmm, where do you expose fetch in util? You expose fetcher as default but that is not equal fetch. Did you try to see a run detail log? that should IMO fail with that code, since it will throw an exception converting txt to json